src/DI/DoctrineExtensionsExtension.php

```diff
@@ -13,10 +13,13 @@
 use Gedmo\Translatable\TranslatableListener;
 use Gedmo\Tree\TreeListener;
 use Nette\DI\CompilerExtension;
+use Nettrine\DBAL\DI\DbalExtension;
```

Comment:
This is not in composer deps, is it?

Reply:
No, it's not. :cry: You're right!

I'm using it for const [few lines lower](https://github.com/nettrine/extensions/pull/7/commits/dc831ec7c6db0b6861acc65265a78faea58988b7#diff-6408d50bd89dfaa07bbafbe4fe997be6R21). Souhld I just change const to something like this:

`public cont TAG_NETTRINE_SUBSCRIBER = 'nettrine.subscriber'`

Reply:
Sure.